plugin.rb

```diff
@@ -165,7 +165,8 @@ class ::DevelopmentAuth::FakeDiscourseConnectController < ::ApplicationControlle
       name
     }
     ADVANCED_FIELDS = DiscourseConnectBase::ACCESSORS.map(&:to_s) - SIMPLE_FIELDS
-    FIELDS = SIMPLE_FIELDS + ADVANCED_FIELDS
+    CUSTOM_FIELDS = ::UserField.all.pluck(:id, :name)&.map{|id, name| {"#{name}": "custom.user_field_#{id}"}}&.reduce(:merge!) || {}
```

Comment:
I don't think this should be in a constant - UserFields can change at runtime. Can we move this into a method so it's recalculated for every request. (we don't care about performance in this plugin, so no need to cache)

Reply:
Like 4da3c06 ?